specification/security/resource-manager/readme.md

```diff
@@ -68,6 +68,10 @@ directive:
     where: $.paths
     from: applications.json
     reason: Suppression of OperationsAPI as it doesn't apply to this specific file.
+  - suppress: OperationsAPIImplementation
```

Comment:
remove this suppression - you are currently exposing the integrations operations in the operations endpoint.